patches/chrome-browser-extensions-api-identity-identity_get_auth_token_function.cc.patch

```diff
@@ -1,23 +1,22 @@
 diff --git a/chrome/browser/extensions/api/identity/identity_get_auth_token_function.cc b/chrome/browser/extensions/api/identity/identity_get_auth_token_function.cc
-index b377740b4bd4898e2414a043255e8bb88ec02af4..d2d756a65c65ddef8135c85728a33a08a8620055 100644
+index b425e031d9153d6df7ead1a9551325703aa92a26..2ba0b5801e0b642dc1b39c2326786e6e774c403e 100644
 --- a/chrome/browser/extensions/api/identity/identity_get_auth_token_function.cc
 +++ b/chrome/browser/extensions/api/identity/identity_get_auth_token_function.cc
-@@ -45,6 +45,14 @@
- #include "google_apis/gaia/gaia_urls.h"
- #include "services/network/public/cpp/shared_url_loader_factory.h"
+@@ -44,6 +44,13 @@
+ #include "google_apis/gaia/gaia_constants.h"
+ #endif
  
 +#include "base/compiler_specific.h"
```

Comment:
this was supposed to be applying an upstream patch as-is per https://github.com/brave/brave-core/pull/14069 so I think there's something wrong here. cc @spylogsster

Reply:
yes, we need this and it works well now, ++